net.sf.eclipsecs.ui/src/net/sf/eclipsecs/ui/CheckstyleMarkerImageProvider.java

```diff
@@ -29,7 +29,7 @@
 
 /**
  * Image provider for Checkstyle markers.
- * 
+ *
```

Comment:
Sounds like auto formatting.
Is it possible to auto format whole project ones to avoid  such extra changes in future?

Reply:
Yes, that's why I have it quite high in the bullet point list of #388. I also don't like reviews that contain whitespace and formatting changes.